NetworkPkg/HttpDxe/HttpsSupport.c

```diff
@@ -631,11 +631,14 @@ TlsConfigureSession (
   //
   HttpInstance->TlsConfigData.ConnectionEnd = EfiTlsClient;
   HttpInstance->TlsConfigData.VerifyMethod = EFI_TLS_VERIFY_PEER;
+  HttpInstance->TlsConfigData.VerifyHost.Flags    = EFI_TLS_VERIFY_FLAG_NO_WILDCARDS;  // MU_CHANGE - Proposed fixes for TCBZ960, invalid domain name (CN) accepted.

```

Comment:
Don't agree with this policy.  We have a use case for using certs that leverage wildcards.   This policy needs to be configurable in a future revision for now i propose EFI_TLS_VERIFY_FLAG_NONE